graph/patterns/dictionary.md

```diff
@@ -175,7 +176,10 @@ PATCH https://graph.microsoft.com/v1.0/users/10/roles/author
 #### Delete an entry from the dictionary
 
 ```HTTP
-DELETE https://graph.microsoft.com/v1.0/users/10/roles/author
+PATCH https://graph.microsoft.com/v1.0/users/10/roles/author
+{
+  "domain": null
+}
 ```
 
 ### CDSL example
```

Comment:
Working with a developer today, I realized that this example is missing how `assignedRoleGroupDictionary` is modeled in the context of the `role` entity type.

Reply:
Yes.

Originally in this PR I tried to make the minimal change to the existing guidance to use PATCH rather than POST.  However, comments have demonstrated that there are many other issues with the guidance, so I am in the process of rewriting it to address some of those more systemic issues.